core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/migrate/MigrationData.java

```diff
@@ -59,10 +55,6 @@ void debug(Supplier<String> supplier) {
         return timer;
     }
 
-    @NotNull Map<Currency, Currency> migratedCurrencies() {
```

Comment:
how come can we no longer check which currencies were migrated?

Reply:
currencies are not migrated since they're not per economy provider anymore, but are global. we migrate the balances of the currencies.